sdk/Cargo.toml

```diff
@@ -36,18 +36,28 @@ features = [
     "pdf",
     "v1_api",
     "diagnostics",
+    "http_ureq",
+    "http_reqwest",
 ]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [features]
-default = ["openssl"]
+# The reason we use `ureq` instead of `reqwest_blocking` is because the blocking version
```

Comment:
What is the basis of having so many options and not just one?

Reply:
> What is the basis of having so many options?

You only need to enable the features for the networking libraries you use.

> Is the resolver used in all feature combinations?

It can be, let's say for instance there are multiple sub-dependencies that use c2pa-rs with different http features, cargo will combine them all. You also may not use async but do use sync, so no reason to enable the async library. In our case, we must use both `ureq` (sync) and `reqwest` (async) because of how our tests are setup.

> It would be nice if the features mapped across all targets, rather than target specific features.

Well they are library-specific, e.g. `http_wasi` enables the `wasi` library but it just so happens it can only be used on WASI. Same with `http_wstd` and the `wstd` library.

Reply:
Mild concern about the test matrix here. What is the minimum set of combinations we need to think of to cover this with enough testing so we feel good here?

Reply:
I am not necessarily a fan of that, because users could still pick and choose. We should aime when possible for less features, not more. But here it might not be possible. Therefore: Ok.

Reply:
With the changes to release-readiness, I'm :+1:.